src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -98,12 +99,80 @@ void UprobesUnwindingVisitor::visit(CallchainSamplePerfEvent* event) {
   CHECK(listener_ != nullptr);
```

Comment:
maybe extract this of most of this function to something that is unit-testable?

Reply:
I am kind of interested how this function react to a tail call landing to a leaf function.

Reply:
Was also thinking of this, so I agree. I'll look into this. Also might look once more into tail-calls.